### ISSUE TITLE
Can the script be run from anywhere?

### ISSUE THREAD

**Issue by khoriba** [deprekate/PHACTS#2]
Hi,
I used PHACTS in my local environment now. Normally, the working directory is the PHACTS directory.
Next, I would like to use it in our batch system. In that case, I can't run `phacts.pl` in the PHACTS directory. 
How can I use PHACTS in my job script in that case?

**Comment by deprekate** [comment]
Are you using the original PERL code or the new python3 code?  

If the PHACTS directory is not on the path, you can either add the directory to your path via:
```
export PATH=$PATH:$HOME/pathto/phacts
```
or run phacts using the full path:
```
~/pathto/phacts/phacts.pl --classes ~/pathto/phacts/classes_lifestyle
```

The new python version should be put in your python bin directory (which might not be on your path) and also does not need the classes_lifestyle file to be specified

**Comment by khoriba** [comment]
Thank you for your reply.
I get src from https://edwards.sdsu.edu/PHACTS/faq.php
and, when I use the full path, I got the following error.
```Could not open file No such file or directory at /home/local/PHACTS/phacts.pl line 100.```

**Comment by deprekate** [comment]
That error (line 100) means the --classes file is not found. Something is wrong the file paths.

Alternatively, you can try the new python version.  It's much easier to use, since the classes file is wrapped internally and automatically used.
```
pip install phacts --user
phacts.py tests/NC_001416.faa
```

**Comment by khoriba** [comment]
I failed to install PHACTS by pip.
```Traceback (most recent call last):
  File "/home/miniconda3/envs/phage/bin/pip", line 7, in <module>
    from pip._internal.cli.main import main
  File "/home/miniconda3/envs/phage/lib/python3.7/site-packages/pip/_internal/cli/main.py", line 9, in <module>
    from pip._internal.cli.autocompletion import autocomplete
  File "/home/miniconda3/envs/phage/lib/python3.7/site-packages/pip/_internal/cli/autocompletion.py", line 10, in <module>
    from pip._internal.cli.main_parser import create_main_parser
  File "/home/miniconda3/envs/phage/lib/python3.7/site-packages/pip/_internal/cli/main_parser.py", line 8, in <module>
    from pip._internal.cli import cmdoptions
  File "/home/miniconda3/envs/phage/lib/python3.7/site-packages/pip/_internal/cli/cmdoptions.py", line 23, in <module>
    from pip._internal.cli.parser import ConfigOptionParser
  File "/home/miniconda3/envs/phage/lib/python3.7/site-packages/pip/_internal/cli/parser.py", line 12, in <module>
    from pip._internal.configuration import Configuration, ConfigurationError
  File "/home/miniconda3/envs/phage/lib/python3.7/site-packages/pip/_internal/configuration.py", line 27, in <module>
    from pip._internal.utils.misc import ensure_dir, enum
  File "/home/miniconda3/envs/phage/lib/python3.7/site-packages/pip/_internal/utils/misc.py", line 42, in <module>
    from pip._internal.locations import get_major_minor_version, site_packages, user_site
  File "/home/miniconda3/envs/phage/lib/python3.7/site-packages/pip/_internal/locations/__init__.py", line 13, in <module>
    from . import _distutils, _sysconfig
  File "/home/miniconda3/envs/phage/lib/python3.7/site-packages/pip/_internal/locations/_distutils.py", line 19, in <module>
    from .base import get_major_minor_version
  File "/home/miniconda3/envs/phage/lib/python3.7/site-packages/pip/_internal/locations/base.py", line 15, in <module>
    site_packages: typing.Optional[str] = sysconfig.get_path("purelib")
  File "/home/miniconda3/envs/phage/lib/python3.7/sysconfig.py", line 521, in get_path
    return get_paths(scheme, vars, expand)[name]
  File "/home/miniconda3/envs/phage/lib/python3.7/sysconfig.py", line 511, in get_paths
    return _expand_vars(scheme, vars)
  File "/home/miniconda3/envs/phage/lib/python3.7/sysconfig.py", line 172, in _expand_vars
    _extend_dict(vars, get_config_vars())
  File "/home/miniconda3/envs/phage/lib/python3.7/sysconfig.py", line 559, in get_config_vars
    _init_posix(_CONFIG_VARS)
  File "/home/miniconda3/envs/phage/lib/python3.7/sysconfig.py", line 430, in _init_posix
    _temp = __import__(name, globals(), locals(), ['build_time_vars'], 0)
ModuleNotFoundError: No module named '_sysconfigdata_x86_64_conda_linux_gnu'
```
I also tried to install from github, but the error occurres.
```
$ python3 setup.py install
Traceback (most recent call last):
  File "setup.py", line 47, in <module>
    main()
  File "setup.py", line 22, in main
    version = get_version(),
  File "setup.py", line 15, in get_version
    with open("VERSION", 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'VERSION'
```

**Comment by deprekate** [comment]
Hm, the conda error seems to be something wrong with the conda install.  I found [this](https://stackoverflow.com/questions/68261254/conda-error-sysconfigdata-x86-64-conda-linux-gnu) mention of the error on stackoverflow.

The error about the version was my fault, I forgot to add the version file to the git repo. That error should be fixed.
However you will have to have fasta35 available on the path, for phacts to work.  I have not gotten around to recoding FASTA35 as a Python C Extension, that installs automatically, but I have included pre compiled binaries for OSX and Linux.  You would just copy the appropriate one to your bin folder (or any folder on your path).
